pandas/tests/io/parser/test_c_parser_only.py

```diff
@@ -575,6 +575,8 @@ def test_bytes_exceed_2gb(c_parser_only):
     if parser.low_memory:
         pytest.skip("not a low_memory test")
 
+    # csv takes 10 seconds to construct, spikes memory to 8GB+, the whole test
```

Comment:
Sidenote: This seems to be more appropriate as an ASV benchmark using `peak_mem`